en/docs/updates/wum-decommission.md

```diff
@@ -19,7 +19,7 @@ recommends all users to shift to the new WSO2 Updates 2.0 model to alleviate fro
 
 
 !!! Note
-    <strong>To all the existing WUM/In-place users:</strong>The Initial WUM/In-place Decommission date has further extended to 7th of August 2021. [Migrate your WSO2 update mechanism to Update 2.0](../../updates/migrating-to-updates2.0/) without fail at your earliest.
+    <strong>To all the existing WUM/In-place users:</strong>The final WUM/In-place decommission date is on 07th March 2023. [Migrate your WSO2 update mechanism to Update 2.0](../../updates/migrating-to-updates2.0/) to avoid inconvenience.
```

Comment:
Shall we add `to avoid any inconvenience` WDYT?

Reply:
Done